src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilderRow.tsx

```diff
@@ -0,0 +1,247 @@
+            value: newValue
+        });
+    };
+
+    const onChangeDropdownValue = (
+        _event: React.FormEvent<HTMLDivElement>,
+        option?: IDropdownOption<any>,
+        _index?: number
+    ) => {
+        onChangeValue(rowId, {
+            combinator: selectedCombinator,
+            operatorData: selectedOperator.data,
+            property: selectedProperty.text, // MAYBE CHANGE TO DATA VALUE?
+            value: option.text
+        });
+    };
+
+    const renderValueField = () => {
+        if (!selectedProperty) {
+            return (
+                <TextField
+                    styles={classNames.subComponentStyles.textfield}
+                    disabled={true}
+                />
+            );
+        } else if (selectedProperty.data.type === 'string') {
+            return (
+                <TextField
+                    styles={classNames.subComponentStyles.textfield}
+                    onChange={onChangeValueField}
+                />
+            );
+        } else if (selectedProperty.data.type === 'boolean') {
+            return (
+                <Dropdown
+                    options={[
+                        {
+                            key: 'True',
+                            text: 'True',
+                            selected: true
+                        },
+                        {
+                            key: 'False',
+                            text: 'False'
+                        }
+                    ]}
+                    onChange={onChangeDropdownValue}
+                    styles={classNames.subComponentStyles.textfield}
+                />
+            );
+        } else {
+            return (
+                <SpinButton
+                    styles={classNames.subComponentStyles.textfield}
+                    onChange={onChangeValueField}
+                />
+            );
+        }
+    };
+
+    return (
+        <div className={classNames.root}>
+            <Stack horizontal={true} style={{ alignItems: 'end' }}>
+                <div className={classNames.lastColumn}>
+                    {position !== 0 && (
+                        <Dropdown
+                            options={[
+                                {
+                                    key: 'And',
+                                    text: 'And',
+                                    selected: true
+                                },
+                                {
+                                    key: 'Or',
+                                    text: 'Or'
+                                }
+                            ]}
+                            onChange={onChangeCombinator}
+                            styles={classNames.subComponentStyles.andDropdown}
+                        />
+                    )}
+                </div>
+                {/* Property */}
+                <div className={classNames.propertyContainer}>
```

Comment:
Can we style the dropdown instead of creating a container?

Reply:
Updated to not use dropdown, now styling is in the component not the container